meetup/services/meetup_info_fetch.py

```diff
@@ -1,10 +1,10 @@
 from django.db import models
 from meetup.services.meetup_api_lookup_agent import MeetupApiLookupAgent
 import time
-import pdb
 
 class MeetupInfoFetch():
```

Comment:
The natural instinct is to give classes noun-y names, but it's OK to use verb-y names, too. This might better be called a `FetchMeetupInfo`. Sounds a little weird at first, but makes sense after a while.